packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -300,8 +308,9 @@ export class SpinButton extends BaseComponent<ISpinButtonProps, ISpinButtonState
    * Increment function to use if one is not passed in
    */
   private _defaultOnIncrement = (value: string): string | void => {
-    let newValue: number = Math.min(Number(value) + Number(this.props.step)!, this.props.max!);
-    newValue = precisionRound(newValue, this.state.precision);
+    const props = this.props as ISpinButtonProps & DefaultProps;
```

Comment:
const { max, steps } = this.props as ISpinnButtonProps & DefaultProps

then you don't need the `!` use in line 312